.github/scripts/check_config_changes_ci.py

```
@@ -0,0 +1,429 @@
+            if r.additions:
+                print(f"     Added:   {', '.join(r.additions)}")
+            if r.removals:
+                print(f"     Removed: {', '.join(r.removals)}")
+            for c in r.changes:
+                print(f"     {c}")
+        else:
+            print(f"  –  {r.label}: no changes")
+ 
+    note = build_pr_note(results)
+    if not note:
+        print("\nℹ️  No notable changes found across all checkers.")
```

Ruff (0.15.9)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)